Libraries/WNScripting/CMakeLists.txt

```diff
@@ -156,6 +156,15 @@ if (NOT WN_DISABLE_CXX)
         COMMAND ${CMAKE_COMMAND} --build . ${WN_LLVM_ADDITIONAL_BUILD_FLAGS} --target ${WNScriptingLLVMLibs}
       WORKING_DIRECTORY ${WNFramework_BINARY_DIR}/Externals/llvm-project/llvm)
 
+  set_target_properties(
```

Comment:
Should this go into the multibuild overlay or windows overlay?

Reply:
Probably, I wasnt sure where to put it